index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Embedding images and videos</title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+	<nav id="navbar">
+	<ul>
+		<li><a href="#"><img  src="menu.PNG"></a></li>
+		<li><a href="#"><img src="youtube.PNG"></a></li>
+		<li id="search"><input id="input" type="text" placeholder="Search" width="400px" margin-right="40px"><button id="search-button"><img id="button" src="search.PNG" ></button></li></a></li>
```

Comment:
Attribute `width` and `margin-right` not allowed on element `input`.
No opening tag for `a` exists also `li` tag is closed twice.